src/renderer/components/multimachinebackups/CloseButtonWithStore.scss

```
@@ -0,0 +1,6 @@
+.erroredState {
+	position: absolute;
+    z-index: 99;
+    right: 30px;
+    top: 100px !important;
```

tab instead of spaces here

```suggestion
    position: absolute;
    z-index: 99;
    right: 30px;
    top: 100px !important;
```